### DISCUSSION TITLE
How new packages are added?

### DISCUSSION THREAD

**Discussion by dutty5** [dd010101/vyos-jenkins#42] [General]
Hello,

I see that VyOS has added a new package to build: [podman](https://github.com/dd010101/vyos-build/commit/c8de52265285b64a51e47a63cbfbe8cdad2b796c).

Does it require to update the corresponding scripts in the vyos-jenkins repo? For example, this file:
[project-jobs.json](https://github.com/dd010101/vyos-jenkins/blob/master/jobs/project-jobs.json)

**Comment by dd010101** [comment]
I'm waiting until they merge [the fix](https://github.com/vyos/vyos-build/pull/721) for the podman, currently the podman build fails anyway, that's why it wasn't added yet.

**Comment by GurliGebis** [comment]
After new packages has been added, running step 6 and 7 will build them 🙂
